clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -93,6 +93,8 @@ def err_fe_backend_error_attr :
 def warn_fe_backend_warning_attr :
   Warning<"call to '%0' declared with 'warning' attribute: %1">, BackendInfo,
   InGroup<BackendWarningAttributes>;
+def warn_toc_unsupported_type : Warning<"The -mtocdata option is ignored "
```

Comment:
Please check the existing diagnostic style. They do not use capitalization or trailing period.
"the" is often omitted.

-mtocdata is ignored for ... because ...